ApollosProject/apollos-apps
# feat: Onboard only seen slides

## DESCRIPTION

### What does this PR do, or why is it needed?

Gives the templates the ability to define what slides to show to users by defining versions. Users will have a `version` stored in async storage. If there are new slides with a higher version than your stored, version you are shown that slide after auth / when opening the app. 

### How do I test this PR?

Testing this is a pain, so I added some helpers

1. Login / Open the app. You should see the new suggested followers slide. 
2. Close / open the app, you should see no new slides.
3. Go to the test utilities tab and reset your onboarding seen status (click the second button). Now close and open the app. You should see the full onboarding.

![2021-03-24 14 34 31](https://user-images.githubusercontent.com/1637694/112365769-67e21500-8cae-11eb-8fb3-4d730c64189b.gif)
![2021-03-24 14 34 10](https://user-images.githubusercontent.com/1637694/112365771-6a446f00-8cae-11eb-8298-53125e8e984f.gif)

You'll need this companion PR: https://github.com/ApollosProject/apollos-templates/pull/312
